### DISCUSSION TITLE
Netherite block

### DISCUSSION THREAD

**Discussion by opekope2** [opekope2/StaffMod#48] [Staff blocks]
Netherite block staff can be seen in Lucky Block Staff at [0:15](https://youtu.be/vlVUVsx6BYY?t=15). It can destroy massive amounts of blocks. If Green hits the 2nd block above ground, the staff can break that block, at least 8 blocks above it (the brick wall is 10 blocks high), 2 blocks below it (at least 3 blocks starting at the 2nd block away from Green all the way through the 10th block, then it's 2 blocks again for at least 2 blocks).
It can also be seen in Monster School at [16:08](https://youtu.be/SNuJrSP3JDA?t=968). Red attacks King Orange, deals massive amount of knockback, and destroys blocks in the direction King Orange flies.

**Comment by TeboMC** [comment]
With enchantments, you could increase the knockback and/or maybe the block destroying radius, with the knockback enchant, you could make the netherite block staff's knockback go higher. or with custom enchantments.

**Reply by opekope2** [reply]
I tried to code canonically accurate knockback, but Minecraft said no.

**Reply by opekope2** [reply]
Another suggestion from a friend that I should make it break blocks in a longer line rather than a cube.

**Reply by opekope2** [reply]
It currently doesn't remove fluids, maybe I can do sth about it (it will either be an enchantment, or netherite-block exclusive (so diamond block won't remove fluids), or tell me if you have a better idea)

**Reply by TeboMC** [reply]
I think you could add a enchant called "Fluid Remover" or somthing similar, there could be 3 levels, level 1 only works on gold, and level 2 works on gold and diamond, and level 3 works on gold, diamond and netherite.
